opl/investigator/config.py

```diff
@@ -75,6 +75,9 @@ def load_config(conf, fp):
         assert not conf.history_es_server.endswith("/")
         conf.history_es_index = data["history"]["es_index"]
         conf.history_es_query = data["history"]["es_query"]
+        conf.history_es_server_user = data["history"]["es_server_user"]
+        conf.history_es_server_pass_env_var = data["history"]["es_server_pass_env_var"]
+        conf.history_es_server_verify = data["history"]["es_server_verify"]
```

Comment:
Make it so that if `es_server_verify` is not configured in the config, `conf.history_es_server_verify` is set to true.